src/WinUI/ArcGIS.WinUI.Viewer/Samples/UtilityNetwork/ValidateUtilityNetworkTopology/ValidateUtilityNetworkTopology.xaml

```diff
@@ -0,0 +1,94 @@
+                        Content="Get State" />
+                <Button x:Name="ValidateBtn"
+                        Grid.Row="0"
+                        Grid.Column="1"
+                        HorizontalAlignment="Stretch"
+                        Click="OnValidate"
+                        Content="Validate" />
+                <Button x:Name="TraceBtn"
+                        Grid.Row="1"
+                        Grid.Column="0"
+                        HorizontalAlignment="Stretch"
+                        Click="OnTrace"
+                        Content="Trace" />
+                <Button x:Name="ClearBtn"
+                        Grid.Row="1"
+                        Grid.Column="1"
+                        HorizontalAlignment="Stretch"
+                        Click="OnClear"
+                        Content="Clear Selected" />
+                <TextBlock x:Name="Status"
+                           Grid.Row="2"
+                           Grid.ColumnSpan="2"
+                           Text="Loading sample..." />
+                <ProgressBar x:Name="IsBusy"
+                             Grid.Row="3"
+                             Grid.ColumnSpan="2"
+                             Height="15"
+                             IsIndeterminate="True"
+                             Visibility="Visible" />
+            </Grid>
+        </Border>
+        <Border x:Name="AttributePicker"
+                HorizontalAlignment="Center"
+                VerticalAlignment="Center"
+                Style="{StaticResource BorderStyle}"
+                Visibility="Collapsed">
+            <Grid>
+                <Grid.RowDefinitions>
+                    <RowDefinition Height="Auto" />
+                    <RowDefinition Height="Auto" />
+                    <RowDefinition Height="Auto" />
+                </Grid.RowDefinitions>
+                <Grid.ColumnDefinitions>
+                    <ColumnDefinition />
+                    <ColumnDefinition />
+                </Grid.ColumnDefinitions>
+                <TextBlock x:Name="FieldName" Grid.Row="0" />
+                <ComboBox x:Name="Choices"
+                          Grid.Row="0"
+                          Grid.Column="1">
+                    <ComboBox.ItemTemplate>
+                        <DataTemplate>
+                            <TextBlock Text="{Binding Name}" />
+                        </DataTemplate>
+                    </ComboBox.ItemTemplate>
+                </ComboBox>
+                <Button Grid.Row="1"
+                        Grid.ColumnSpan="2"
+                        Click="OnApplyEdits"
+                        Content="Update and Apply Edit" />
+                <Button x:Name="CloseButton"
+                        Grid.Row="2"
+                        Grid.ColumnSpan="2"
+                        Click="OnClear"
+                        Content="Cancel" />
+            </Grid>
+        </Border>
+        <Grid x:Name="Dialog" />
```

Comment:
What is this dialog used for? If for error messages, I think we use `MessageDialog2`

Reply:
https://learn.microsoft.com/en-us/uwp/api/windows.ui.popups.messagedialog?view=winrt-22621
Microsoft recommends ContentDialog. I will move to MessageDialog2 since our code sample convention is to use our own helper method.